aztk/utils.py

```diff
@@ -0,0 +1,210 @@
+import getpass
+import time
+from typing import List
+from subprocess import call
+import azure.batch.models as batch_models
+import aztk_sdk.spark
+import aztk_sdk.utils.error as error
+import aztk_sdk.utils.get_ssh_key as get_ssh_key
+from . import log
+
+
+def get_ssh_key_or_prompt(ssh_key, username, password, secrets_config):
+    ssh_key = get_ssh_key.get_user_public_key(ssh_key, secrets_config)
+
+    if username is not None and password is None and ssh_key is None:
+        password = getpass.getpass("Please input a password for user '{0}': ".format(username))
+        confirm_password = getpass.getpass("Please confirm your password for user '{0}': ".format(username))
+        if password != confirm_password:
+            raise error.AztkError("Password confirmation did not match, please try again.")
+        if not password:
+            raise error.AztkError(
+                "Password is empty, cannot add user to cluster. Provide a ssh public key in .aztk/secrets.yaml. Or provide an ssh-key or password with commnad line parameters (--ssh-key or --password).")
+    return ssh_key, password
+
+def print_cluster(cluster: aztk_sdk.spark.models.Cluster, batch_client):
+    node_count = __pretty_node_count(cluster)
+
+    log.info("")
+    log.info("Cluster         %s", cluster.id)
+    log.info("------------------------------------------")
```

Comment:
print format "-<36"?

Reply:
I'm confused about what you are asking here. Line 38 & 39 handle alignment.